R/defaults.R

```diff
@@ -76,14 +63,16 @@ epidist_stancode.default <- function(data, ...) {
 #' @family defaults
 #' @export
 epidist.default <- function(data, formula = brms::bf(mu ~ 1, sigma ~ 1),
-                            family = brms::lognormal(), fn = brms::brm, ...) {
+                            family = brms::lognormal(), prior = NULL,
+                            fn = brms::brm, ...) {
   epidist_validate(data)
   epidist_family <- epidist_family(data, family)
   epidist_formula <- epidist_formula(
     data = data, family = epidist_family, formula = formula
   )
+  class(family) <- c(class(family), family$family)
```

Comment:
Arguably this should happen inside the `epidist_prior` function. Easy to forget to do it outside

Reply:
yes agree

Reply:
Done